test/parallel/test-tls-server-failed-handshake-emits-clienterror.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const common = require('../common');
+
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
+const tls = require('tls');
+const net = require('net');
+const assert = require('assert');
+
+const bonkers = Buffer.alloc(1024, 42);
```

Comment:
maybe a comment, "Construct a buffer that is not a valid TLS handshake.", or just change var name -
 `s/bonkers/invalidTlsHandshake/`?